crates/topos-tce-storage/src/types.rs

```diff
@@ -56,7 +56,7 @@ pub struct CheckpointSummary {
     sequence_number: usize,
     checkpoint_data: Vec<SourceStreamPosition>,
 }
-pub struct VerifiedCheckpointSummary(CheckpointSummary, ValidatorQuorumSignatureInfo);
+pub struct VerifiedCheckpointSummary((), ());
```

Comment:
Seems like these are not being used at all @Freyskeyd. Is there a reason for it?